src/elements/content-sidebar/README.md

```diff
@@ -61,7 +61,12 @@ var ContentSidebar = require('./ContentSidebar').default;
 #### MetadataSidebarProps
 | Prop | Type | Default | Description |
 | --- | --- | --- | --- |
-| *tbd* | - | - | - |
+| selectedTemplateKey | `string`         |  | `templateKey` for the template to show. If provided, |
+|                     |                  |  | no other templates will be visible. |
+| templateFilters     | `Array<string>`  |  | Template field IDs for fields that should be visible and editable. |
```

Comment:
Could also be simplified to just accept an array of strings, if that makes things easier.